install-usercss/install-usercss.js

```diff
@@ -16,7 +16,8 @@
   let port;
 
   if (params.has('direct')) {
-    $('.live-reload').textContent = t('liveReloadUnavailable');
+    $('.live-reload input').classList.add('hidden');
```

Comment:
Can we fix this using CSS?
```css
.live-reload .unavailable-message {
	display: none;
}
.live-reload.unavailable .unavailable-message {
	display: inline;
}
.live-reload.unavailable input {
	display: none;
}
```
So we just have to toggle `unavailable` class in js.

Reply:
Yes, that's a much cleaner solution! 😸